src/utils.js

```diff
@@ -4,11 +4,19 @@ const marketApi = axios.create({
     baseURL: 'https://nc-news-40gp.onrender.com/api'
 })
 
-export const fetchArticles = () => {
-    return marketApi.get('/articles')
+export const fetchArticles = (topic = '') => {
+    const queryParams = [];
+    let queryString = "";
+    if (topic){
+        queryParams.push(`topic=${topic}`)
+    }
+    if (queryParams.length > 0){
+        queryString = "?"+queryParams.join('&');
+    }
+    return marketApi.get(`/articles${queryString}`)
```

Comment:
Updating the url to deal with queries is OK, but axios has its own way to do it - get method can take an object as the second argument, it'll make it a bit easier
e.g. `marketApi.get("/articles", {params:{topic, sort_by, ...}})`

Reply:
Also it would be better if the name 'marketApi' can be more news related